.env.example

```diff
@@ -1,3 +1,6 @@
 VITE_API_BASE_URL=
 VITE_FRONTEND_VERSION=
 VITE_FEATURE_OS_KEY=
+
+VITE_ZENML_DEBUG=
+VITE_ZENML_ANALYTICS_OPT_IN=
```

Comment:
Pending add this vars to off telemetry

Reply:
I'm not sure if they're having an effect, because they're having the `VITE_` prefix, and as far as I can see they are not used inside the code right? @schustmi What would you say, I think if the env-variable is not part of the code I would not put it inside the .env.example. Would you encourage contributors to turn it off, I feel like mentioning it in the readme, or linking to the documentation should do the trick 🤔

Reply:
What is that `VITE_` prefix, can't we just remove it.

I think having it in the readme is fine but lots of people will probably ignore it. If there is an automated way like having it in this example file, that seems to make more sense to me. Or even better, having it here and mentioning it in the docs.

Reply:
Only the variabels that are prefixed with `VITE_` are available for the build tool so no other env variable is accidentally leaked to the frontend code. 

To be honest, I dont even know if the variable has an effect if we put it to the file, because the variable needs to be there when `zenml up` is run, and if this happens before the dev server is started I think it's not in the environment yet. 

Plus it doesnt apply to the docker way right? To make it work with docker it needs to be set in the docker-run command doesnt it?

Reply:
Yep that's all correct. If this is just for frontend env, it's not necessary

Reply:
I will move this to a new PR so as not to block this PR @schustmi @Cahllagerfeld